.github/labeler.yml

```diff
@@ -1,35 +1,61 @@
 area/api:
-  - registry/api
-  - registry/handlers
+  - changed-files:
+    - any-glob-to-any-file:
+      - registry/api/**/*
+      - registry/handlers/**/*
 area/auth:
-  - registry/auth
+  - changed-files:
+    - any-glob-to-any-file:
+      - registry/auth/**/*
 area/build:
-  - Makefile
-  - Dockerfile
-  - docker-bake.hcl
-  - dockerfiles/**/*
+  - changed-files:
+    - any-glob-to-any-file:
+      - Makefile
+      - Dockerfile
+      - docker-bake.hcl
+      - dockerfiles/**/*
 area/cache:
-  - registry/storage/cache
+  - changed-files:
+    - any-glob-to-any-file:
+      - registry/storage/cache/**/*
 area/ci:
-  - .github/**/*
-  - tests
-  - testutil
+  - changed-files:
+    - any-glob-to-any-file:
+      - .github/**/*
+      - tests/**/*
+      - testutil/**/*
 area/config:
-  - configuration
+  - changed-files:
+    - any-glob-to-any-file:
+      - configuration/**/*
 area/docs:
-  - README.md
-  - docs/**/*.md
+  - changed-files:
+    - any-glob-to-any-file:
+      - README.md
+      - docs/**/*.md
 area/proxy:
-  - registry/proxy
+  - changed-files:
+    - any-glob-to-any-file:
+      - registry/proxy/**/*
 area/storage:
-  - registry/storage
+  - changed-files:
+    - any-glob-to-any-file:
+      - registry/storage/**/*
 area/storage/azure:
-  - registry/storage/azure
```

Comment:
`registry/storage/azure, registry/storage/gcp, registry/storage/s3-aws` don't exist I think the intention was to reference the driver directories. Correct me if I got this wrong

Reply:
Yeah, that's correct